extensions/openwsman/processors/SourceInitiatedSubscriptionListener.cpp

```diff
@@ -807,10 +806,11 @@ void SourceInitiatedSubscriptionListener::onSchedule(const std::shared_ptr<core:
   if (!context->getProperty(SSLCertificateAuthority.getName(), ssl_ca_file)) {
     throw Exception(PROCESSOR_EXCEPTION,"SSL Certificate Authority attribute is missing");
   }
+  bool verify_peer = true;
```

Comment:
This seems a bit superfluous. Instead declaring `verify_peer` outside of the else scope consider removing the introduced block:
```c++
if(missing) {
  throw ...;
}
bool verify_peer = ...
```